ui/src/app/pages/data/cohort/attributes-page.spec.tsx

```diff
@@ -75,51 +70,55 @@ describe('AttributesPageV2', () => {
     };
   });
 
-  it('should render', () => {
-    const wrapper = component();
-    expect(wrapper.exists()).toBeTruthy();
+  it('should render', async () => {
+    component();
+
+    expect(screen.getByText(/number of participants:/i)).toBeTruthy();
   });
 
   it('should not call api and render a single dropdown for Height in Physical Measurements', async () => {
-    const wrapper = component();
-    await waitOneTickAndUpdate(wrapper);
+    await component();
     expect(mockCountParticipants).toHaveBeenCalledTimes(0);
     expect(mockFindCriteriaAttributeByConceptId).toHaveBeenCalledTimes(0);
     expect(mockFindSurveyVersionByQuestionConceptId).toHaveBeenCalledTimes(0);
     expect(
       mockFindSurveyVersionByQuestionConceptIdAndAnswerConceptId
     ).toHaveBeenCalledTimes(0);
-    expect(wrapper.find('div[id="numerical-dropdown-0"]').length).toBe(1);
-    expect(wrapper.find('div[id="numerical-dropdown-1"]').length).toBe(0);
-    expect(wrapper.find('[id="numerical-input-0-0"]').length).toBe(0);
-    expect(
-      wrapper.find('[data-test-id="attributes-add-btn"]').first().props()
-        .disabled
-    ).toBeFalsy();
+    screen.getByRole('button', {
```

Comment:
could you clarify how this corresponds to the previous version?

Reply:
Good question, this button is the down arrow on the far right of the dropdown. 
It has an aria-label, so it seemed like the easiest/,ost descriptive thing to select on:
<img width="1390" alt="image" src="https://github.com/all-of-us/workbench/assets/24514630/96b5ab7e-c282-46c3-8424-405503f6bf93">

Reply:
Is this the same element as numerical-dropdown-0?

Reply:
<img width="1236" alt="image" src="https://github.com/all-of-us/workbench/assets/24514630/03a195b3-ae5e-46f6-8b68-d2c51aa9604b">

Reply:
ah now I see it one level up in the DOM as well, great